### PR TITLE
Added support for python 3.x. Fixes #3

### LINKED ISSUE #3: Import not working on Windows, Python 3.4
```
pip install commentjson
python
>>> import commentjson
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\PyEnvs\3.4\lib\site-packages\commentjson\__init__.py", line 1, in <module>
    from commentjson import dump
ImportError: cannot import name 'dump'
>>>
```

### PR DESCRIPTION
**Pull request by esafak**
It was not working in OS X either, so I took the liberty of making a few changes without breaking backward compatibility.

p.s.: @vaidik might like to set up [automatic deployment to pypi](http://docs.travis-ci.com/user/deployment/pypi/).

### CODE REVIEW

File: commentjson/__init__.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
-from commentjson import dump
-from commentjson import dumps
-from commentjson import JSONLibraryException
-from commentjson import load
-from commentjson import loads
+__all__ = ['dump', 'dumps', 'load', 'loads', 'JSONLibraryException']
+
+try: # for python 2.x
+	from commentjson import dump, dumps, load, loads, JSONLibraryException
```
Comment (vaidik) [inline commentjson/__init__.py]: I think this line is not required at all, we can do the import for both 2.x and 3.x as on line 6, no?

**Comment by vaidik** [comment]
@esafak thanks for taking out time to fix this. I really appreciate it. Left just one comment. Looks good otherwise.

File: commentjson/__init__.py (hunk 1 above)
Comment (esafak) [inline commentjson/__init__.py]: Could be; I'm not able to test now. I just know that the original version did not work.

**Comment by gabyx** [comment]
please merge this :-)

**Comment by mverleg** [comment]
+1 would be very useful

**Comment by mverleg** [comment]
Ehm nothing changed?

**Comment by esafak** [comment]
@mverleg PR #13 was merged instead.

**Comment by mverleg** [comment]
Afraid it didn't fix anything for me, I cloned just before posting that comment and 

    python3 -c 'import commentjson'

doesn't work for me (but python2 does).

**Comment by esafak** [comment]
I just tested and you seem to be right. @vaidik can take it from here.

**Pull request closed without merging**